examples/create-job.js

```diff
@@ -0,0 +1,43 @@
+
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+
+function getJobNames(nameSpace, kubernetes) {
+  return kubernetes.jobs.list(nameSpace).map(function(job){
+    return job.name
+  })
+}
+
+export default function () {
+  const kubernetes = new Kubernetes({
+    // config_path: "/path/to/kube/config", ~/.kube/config by default
+  })
+  const nameSpace = "default"
+  const newJob = {
+    TypeMeta:{
+      ApiVersion: "batch/v1",
+      Kind: "Job"
+    },
+    ObjectMeta: {
+      Name: "pi"
+    },
+    Spec: {
+      Template: {
+        Spec: {
+          Containers: [
+            { 
+              Name: "pi",
+              Image: "perl",
+              Command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+            }
+          ],
+          restartPolicy: "Never"
+        }
+      },
+      BackoffLimit: 4
+    }
+  }
```

Comment:
Random thoughts for @simskij :smile: 

I'm all in for having a `apply` method that can apply any kind of manifest, not only job-related ones.

For jobs, I expect some kind of "curated" methods/helpers. In xk6-kubernetes-jobs you can create a new job like this:
```
client.create("pi-big","perl","perl -Mbignum=bpi -wle print bpi(2000)")
```

Reply:
Sounds good! @lxkuz, are you ok with this change? Making the API for creating jobs a bit more "tailored", and then we'll have generic `apply`/`delete` functions at a later stage? The signature could be:

```js
  kubernetes.jobs.create("job-name", "image", "command", "namespace (optional, defaults to default)");
```

Reply:
@simskij @dgzlopes  please take a look, I've updated the code